.github/workflows/enforce-evm-migration.yaml

```diff
@@ -0,0 +1,45 @@
+name: Ensure EVM module migration when bytecode changes
+
+on:
+  - pull_request
+
+jobs:
+  ensure-evm-migration:
+    strategy:
+      matrix:
+        go-version:
+          - 1.18
+        os:
+          - ubuntu-18.04
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: '0'
+          ref: ${{ inputs.branch }}
+          submodules: recursive
+
+      - name: Compare bytecode version to previous tagged version
+        run: |
+          tag=$(git tag --list --merged HEAD --sort=-v:refname | head -n 1)
```

Comment:
Maybe put this in a script to keep the workflow cleaner?

Reply:
then you just have to go look in a separate file for it, since there is only a single step I don't see any benefit in splitting this out